river/stats/lossy.py

```diff
@@ -0,0 +1,119 @@
+import math
+import typing
+from operator import itemgetter
+
+from river import stats
+
+
+class LossyCount(stats.base.Univariate):
+    """Lossy Counting with Forgetting factor[^1].
+
+    Keep track of the frequent itemsets in a data stream and apply a forgetting factor to
+    discard previous frequent items that do not often appear anymore.
+
+    Considering a data stream where `N` elements were observed so far, the Lossy Counting
```

Comment:
I would prefer `n` to `N`, but it's your call

Reply:
The paper used `N`, but in this context `n` will do the trick.